src/www/ui/api/documentation/openapi.yaml

```
@@ -3363,6 +3406,46 @@ components:
           type: string
           example:
             "Main group"
+    GetClearingHistory:
+      type: object
+      properties:
+        date:
+          type: string
+          format: YYYY-mm-ddTHH:MM:SS
```

That is not what I meant :-)
```suggestion
          format: date
```